var/spack/repos/builtin/packages/spiner/package.py

```diff
@@ -35,8 +35,8 @@ class Spiner(CMakePackage, CudaPackage):
     variant("format", default=False, description="Clang-Format Support")
 
     depends_on("cmake@3.12:")
-    depends_on("catch2@2.13.4:2.13.6")
-    depends_on("ports-of-call@1.1.0:")
+    depends_on("catch2@2.13.4:2.13.9")
+    depends_on("ports-of-call@1.2.0:")
```

Comment:
Is `1.1.0` still relevant for any of the (earlier) versions?

Reply:
I ask because, if it is, you can add a `when` argument (see https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies) whose value can be a version range.

Reply:
@tldahlgren no, the older version would break in some variants due to a cmake bug. the new version should be used instead.

Reply:
I should've paid more attention to the needed PR....